tasks/main.yml

```diff
@@ -44,6 +44,26 @@
     update_cache: yes
     state: installed
 
+- name: grep for LimitNOFILE in systemd service
```

Comment:
I was suspecting this might get a bit tricky using `lineinfile`.  You should be able to use the [`ini_file` module](http://docs.ansible.com/ansible/ini_file_module.html), though, don't you think?